src/common/components/ModifierBar/ModifierItem/ModifierItem.tsx

```diff
@@ -15,7 +16,7 @@ export const ModifierItem: React.FC<ModifierItemProps> = ({
   modifier,
   onChange,
 }) => {
-  const modifierValue = Math.floor((modifier.total - 10) / 2);
+  const modifierValue = calculateModifierBonus(modifier.total ?? 0);
```

Comment:
This was a re-occurring calculation so I moved it out into a utilities file. So anytime we need to know the modifier bonus we just call that function. I think in total we are doing the same calculation in 3 or 4 sub components. Less room for error.